pkg/build/dir_structure.go

```diff
@@ -0,0 +1,63 @@
+package build
```

Comment:
Not sure whether this is the right package for this. It's also why I kept all fields of the type exported for now. Thoughts?